ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml

```diff
@@ -29,11 +29,19 @@ chain:
       - name: HYPERSHIFT_NAMESPACE
         default: "clusters"
         documentation: "The Namespace where to create the HostedCluster and NodePools"
+      - name: HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT
+        default: "false"
+        documentation: "Whether to use the generic CI account or the HyperShift OSD account for the guest clusters infra. For the infra created for the clusters. E.g. For cluster-bot we use the generic CI account"
     commands: |-
+      AWS_GUEST_INFRA_CREDENTIALS_FILE="/etc/hypershift-ci-jobs-awscreds/credentials"
+      if [[ $HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT == "true" ]]; then
```

Comment:
thanks, please make sure to pass the right domain here as well https://github.com/openshift/release/pull/32635/files